docs/dev/object_storage.md

```diff
@@ -112,3 +112,149 @@ Currently only snapshot backup is possible, so first one needs to take [snapshot
+The structure under this prefix is identical to what you’d find in the typical Scylla snapshot.  
+There is a manifest file which contains the list of Data files for each SSTable, the schema file and all the SSTables  
+components stored flat under the prefix.
+```perl
+scylla-bucket/prefix/
+│
+├── manifest.json
+├── schema.cql
+|
+├── me-3gqe_1lnj_4sbpc2ezoscu9hhtor-big-Data.db
+├── me-3gqe_1lnj_4sbpc2ezoscu9hhtor-Index.db
+├── me-3gqe_1lnj_4sbpc2ezoscu9hhtor-Summary.db
+├── ...
+│
+├── ma-1abx_k29m_9fyug3sdtjwj8krpqh-big-Data.db
+├── ma-1abx_k29m_9fyug3sdtjwj8krpqh-Index.db
+├── ma-1abx_k29m_9fyug3sdtjwj8krpqh-Summary.db
+├── ...
+│
+└── ... (more SSTable components)
+```
+See the API [documentation](#copying-sstables-on-s3-backup) for more details about the actual backup request.
+
+3. `CREATE KEYSPACE` with S3 storage
+
+When creating a keyspace with S3 storage, the data is stored under the bucket passed as argument to the `CREATE KEYSPACE` statement.  
+Once the statement is issued, Scylla will use transparently the S3 bucket as the location of the SSTables for that keyspace.  
+Like in the case above, there is no hierarchy for the data, *all SSTables components are stored flat within the bucket*.
+```perl
+scylla-sstables-bucket/
+│
+├── me-3gqe_1lnj_4sbpc2ezoscu9hhtor-big-Data.db
+├── me-3gqe_1lnj_4sbpc2ezoscu9hhtor-Index.db
+├── me-3gqe_1lnj_4sbpc2ezoscu9hhtor-Summary.db
+├── ...
+│
+├── ma-1abx_k29m_9fyug3sdtjwj8krpqh-big-Data.db
+├── ma-1abx_k29m_9fyug3sdtjwj8krpqh-Index.db
+├── ma-1abx_k29m_9fyug3sdtjwj8krpqh-Summary.db
+├── ...
+│
+└── ... (more SSTable components)
+```
+
+## Downloading, deleting, uploading SSTables
+
+To manually manage sstables on S3, AWS CLI commands can be used, but first it's mandatory to have awscli  
+installed ([installation guide](https://docs.aws.amazon.com/cli/latest/userguide/getting-started-install.html)) and have the proper credentials set up in order to be able to access ScyllaDB S3 buckets.
+
+Please make sure your `~/.aws/credentials` file points to a valid set of S3 credentials.  
+Either refresh credentials if you use an OKTA-based fetching tool or make sure they point to a valid IAM user with S3 access.
+
+Provided all the prerequisites above are fulfilled and you're able to run
+```sh
+aws s3 ls s3://your-bucket/
+```
+and see something (or at least not see an error if the bucket is empty), you're all set for the next commands.
+> **NOTE:**
+> Please refer to the sections above for the prefix layout of each S3 use case.
+
+### Downloading SSTables
+
+Fetching the SSTables of your backup can be easily done by  
+e.g. copying each individual component
+```sh
+aws s3 cp s3://your-bucket/path/to/sstable/me-3gqb_1izi_0pxn421yzymfw5c8zf-big-Data.db  /local/path/to/sstable/component
+```
+or downloading an entire sstable using globs
+```sh
+aws s3 cp s3://your-bucket/path-to-sstables/ /local/path/for/sstables --exclude "*" --include 'some-sstable-generation-big-*' --recursive
+```
```

Comment:
Why do we want to duplicate part of the `aws` tool documentation? I mean -- we don't document how to `ls`, `cp` and `rm` files from local directory, why doing it for s3 storage then?

Reply:
For me it makes sense to have them as not everybody is familiar with awcli or even S3 altogether, so it might come in handy to have the commands available for copy pasta.
@denesb is the requester of this documentation, do you think it makes sense to have the awscli commands listed in this doc?

Reply:
Unlike `ls`, `cp`, `rm` and friends, which we use daily, I have never used `aws s3` before, probably the first time I will use it is when something will go wrong with an SStable stored in S3. Having ready to copy basic examples is great help for such rarely used commands.

Reply:
I don't like having that elaborated documentation on `aws s3 ...` CLI in our docs. Its ls, cp, etc. syntax is a) very well documented in `aws s3 <cmd> help` and b) resembles regular ls, cp, etc. commands. The better approach from my perspective is to have this very text (about `aws s3 ... help` goodness) in docs and that's it.

However, don't count it as veto power on that PR, is other parties agree to have it, I will not object.